Lib/subprocess.py

```diff
@@ -230,7 +230,7 @@ def call(*popenargs, timeout=None, **kwargs):
     """
     with Popen(*popenargs, **kwargs) as p:
         try:
-            return p.wait(timeout=timeout)
+            return p.wait(timeout)
```

Comment:
is there an easy to add keyword argument support?

Reply:
Other than a FromArgs derive on a struct, I don't think so.